src/main/java/oss/fosslight/service/impl/VerificationServiceImpl.java

```diff
@@ -274,6 +275,30 @@ public boolean getNoticeHtmlFile(OssNotice ossNotice, String contents) throws IO
 			return getNoticeVelocityTemplateFile(contents, prjInfo);	
 		}
 	}
+
+	@Override
+	public boolean getReviewReportPdfFile(OssNotice ossNotice) throws IOException {
+		return getReviewReportPdfFile(ossNotice, null);
+	}
+
+	@Override
+	public boolean getReviewReportPdfFile(OssNotice ossNotice, String contents) throws IOException {
+		Project prjInfo = projectService.getProjectBasicInfo(ossNotice.getPrjId());
+
+		// OSS Notice가 N/A이면 고지문을 생성하지 않는다.
+		if(CoConstDef.CD_NOTICE_TYPE_NA.equals(prjInfo.getNoticeType())) {
+			return true;
+		}
+
+		prjInfo.setUseCustomNoticeYn(!isEmpty(contents) ? CoConstDef.FLAG_YES : CoConstDef.FLAG_NO);
+		contents = avoidNull(contents, PdfUtil.getInstance().getReviewReportHtml(ossNotice.getPrjId()));

```

Comment:
fosslight/src/main/java/oss/fosslight/service/impl/VerificationServiceImpl.java:294: error: cannot find symbol
                contents = avoidNull(contents, PdfUtil.getInstance().getReviewReportHtml(ossNotice.getPrjId()));
Please, fix this problem.
getReviewReportHtml method is not defined.

Reply:
@MyunghyunNero will modify code in #736 from `htmlToPdfTemplate` to `getReviewReportHtml` soon